src/tools/altinn-fleet-stats/backend/altinn_fleet/bpmn.py

```
@@ -0,0 +1,212 @@
+        if current in visited:  # cycle guard
+            return
+        visited = visited | {current}
+        node = nodes.get(current)
+        if not node:
+            return
+        if node["kind"] == "task":
+            path = path + [{
+                "id": current,
+                "name": node["name"],
+                "altinn_type": node["altinn_type"],
+            }]
```

Ruff (0.15.12)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)